orca-clouddriver/src/main/java/com/netflix/spinnaker/orca/clouddriver/tasks/monitoreddeploy/MonitoredDeployBaseTask.java

```diff
@@ -137,12 +133,12 @@ public long getDynamicTimeout(StageExecution stage) {
 
     try {
       return executeInternal(stage, monitorDefinition);
-    } catch (RetrofitError e) {
+    } catch (SpinnakerServerException e) {
       log.warn(
           "HTTP Error encountered while talking to {}->{}, {}}",
           monitorDefinition,
           e.getUrl(),
-          getRetrofitLogMessage(e.getResponse()),
+          getRetrofitLogMessage(e),
```

Comment:
It's pretty tempting to use `e.getMessage()` here, or to ditch it completely since we're already logging the entire exception.  This looks dynamic enough (i.e. we're not talking about an endpoint elsewhere in spinnaker) that the headers could actually be helpful.  getRetrofitLogMessage collects status and reason that are already in e.getMessage.  For SpinnakerHttpException, the url is there too.  Perhaps there's some work to do to include the URL in e.getMessage() for SpinnakerServerException.

After staring at this for a while, I'd like to:
* rename getRetrofitLogMessage to something that doesn't have retrofit in the name
* change this method to only return body and headers since status/reason are already handled.

Reply:
- Renamed the method getRetrofitLogMessage() to getErrorMessage().
- Modified the method getErrorMessage() to return the concatenated String values of e.getMessage(), headers and body.